### PR TITLE
Dashboard view redesign

### PR DESCRIPTION
**Pull request by ktkization**

### CODE REVIEW

File: webapp/lib/controller.dart [hunk 1]
```diff
@@ -120,24 +118,24 @@ void loadProjectSelectorView() {
...
   view.navView.projectTitle = project?.projectName;
   view.navView.projectOrganizations = [''];
-  var dashboardView = new view.DashboardView();
+  var dashboardView = new view.DashboardView(model.conversationData);
   dashboardView.activePackages.addAll(
     [
-      new view.ActivePackagesViewPartial('Urgent conversations', '#/conversations', '#/escalates-configuration'),
-      new view.ActivePackagesViewPartial('Open conversations', '#/conversations', '#'),
-      new view.ActivePackagesViewPartial('Batch replies (Week 12)', '', '#/batch-replies-configuration'),
+      new view.ActivePackagesViewPartial('Urgent conversations', '#/conversations', '#/escalates-configuration', '3 awaiting reply'),
```
Comment (marianamarasoiu) [inline webapp/lib/controller.dart]: can we get the number `3` from `conversationData['needs-urgent-intervention']`?

File: webapp/lib/controller.dart [hunk 2]
```diff
@@ -120,24 +118,24 @@ void loadProjectSelectorView() {
...
+      new view.ActivePackagesViewPartial('Open conversations', '#/conversations', '#', '20 open conversations'),
```
Comment (marianamarasoiu) [inline webapp/lib/controller.dart]: this 20 is likely going to be different to the 20 below... let's change it to 30 so as not to confuse things

File: webapp/lib/model.dart [hunk 3]
```diff
@@ -11,6 +11,25 @@ Map<String, String> teamMembers  = {
   'Team Member 3': 'email3@organization.org'
 };
 
+Map conversationData = {
+    'needs-urgent-intervention': 3,
+    'awaiting-reply': 20,
+    'conversations': [
+      {
+        'text': 'Waa inlahelaa daryel cifimad iyo nabad warta nanakashaqeyan bulshada dhaman wanasodhaweyney naa sanad ka nasohayo hadoo allah nakarsiyo',
+        'description': 'Male, 20, Displaced'
```
Comment (marianamarasoiu) [inline webapp/lib/model.dart]: let's use 'demogs' instead of 'description'

File: webapp/web/dashboard.css [hunk 4]
```diff
@@ -1,91 +1,233 @@
...
+  display: flex;
+  flex-direction: column;
+  border: 2px solid #cacaca;
+  padding: 20px;
+}
+
+.project-audience-content-item {
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 1.3em;
```
Comment (marianamarasoiu) [inline webapp/web/dashboard.css]: this is a little too large and dominant. let's make it `1.1em`

File: webapp/web/dashboard.css [hunk 5]
```diff
@@ -1,91 +1,233 @@
...
-.active-packages__package {
-    border-style: solid;
-    border-width: 1px;
-    border-radius: .5rem;
-    border-color: rgba(0, 0, 0, .1);
-    padding: 6px;
-    margin-bottom: 20px;
-    width: 80%
+.project-audience-content-item__text {
+  flex-basis: 100%;
```
Comment (marianamarasoiu) [inline webapp/web/dashboard.css:106]: There's a little bit too much whitespace around these messages, as this element is inheriting the margin of `.project-audience-content-item`. Let's set `margin: 0` here
```suggestion
  flex-basis: 100%;
  margin: 0;
```

File: webapp/web/dashboard.css [hunk 6]
```diff
@@ -1,91 +1,233 @@
...
+}
+
+.active-package-chart__data {}
+
+.active-package-chart__description {
+  margin: 0px;
+}
+
+.available-package-name {
+  font-size: 1.1em;
```
Comment (marianamarasoiu) [inline webapp/web/dashboard.css:188]: let's make this bolder, it looks a bit too much like the rest of the text in the package box:
```suggestion
  font-size: 1.1em;
  font-weight: 600;
```

File: webapp/web/dashboard.css [hunk 7]
```diff
@@ -1,91 +1,233 @@
...
+  align-self: center;
+}
+
+.active-package-title {
+  display: flex;
+  align-self: center;
+}
+
+.active-package-title__text {
+  font-size: 1.1em;
```
Comment (marianamarasoiu) [inline webapp/web/dashboard.css:162]: let's make this bolder to stand out a bit more:
```suggestion
  font-size: 1.1em;
  font-weight: 600;
```

File: webapp/lib/controller.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/controller.dart]: Okay. Changed

File: webapp/lib/controller.dart (hunk 2 above)
Comment (ktkization) [reply webapp/lib/controller.dart]: OKay. Chaged

Comment (ktkization) [reply webapp/lib/controller.dart]: OKay. Changed

File: webapp/lib/model.dart (hunk 3 above)
Comment (ktkization) [reply webapp/lib/model.dart]: Renamed

File: webapp/web/dashboard.css (hunk 4 above)
Comment (ktkization) [reply webapp/web/dashboard.css]: Modified

**Review by marianamarasoiu — APPROVED** [review]
LGTM thanks @ktkization !

**Pull request merged**